src/Simulation/Simulators/QuantumExecutor/QuantumExecutorSimulator.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Quantum.Simulation.Common;
+
+namespace Microsoft.Quantum.Simulation.QuantumExecutor
+{
+    /// <summary>
+    /// Simulator that redirects all the calls to a class implementing <see cref="IQuantumExecutor"/> interface.
+    /// </summary>
+    public partial class QuantumExecutorSimulator : SimulatorBase
+    {
+        private const int PreallocatedQubitCount = 1000;
```

Comment:
Does not matter much. It is only used for QubitManager initialization, and qubit manager is a lean machine that is not wasting too much. It will also grow as necessary. But 256 will be OK for now.

---
In reply to: [348348740](https://github.com/microsoft/qsharp-runtime/pull/82#discussion_r348348740) [](ancestors = 348348740)